docs/Project.toml

```diff
@@ -10,6 +10,8 @@ DynamicalSystems = "61744808-ddfa-5f27-97ff-6e42cc95d634"
 DynamicalSystemsBase = "6e36e845-645a-534a-86f2-f5d4aa5a06b4"
 Entropies = "ed8fcbec-b94c-44b6-89df-898894ad9591"
 FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
+GraphPlot = "a2cc645c-3eea-5389-862e-a155d0052231"
```

Comment:
Can we please avoid installing this and instead just use a saved image of a graph plot? Installing and compiling the Plots library takes a huge amount of time.

Reply:
(Also, the Plots.jl library is in my eyes well inferior to PyPlot.jl in offered functionality, while the Julia community will largely switch to Makie.jl once it is stable)

Reply:
done